client/components/Update/UpdatesPage.vue

```diff
@@ -1,48 +1,39 @@
 <template>
-  <main>
-    <h1>
-      Project: {{ project.projectName }}
-    </h1>
-    <section
-      v-for="teammate in project.participants"
-      class="user-updates"
-    >
-      <h2>{{ teammate }}</h2>
-      <template v-for="update in (updates[teammate] || [])">
-        <UpdatePreview :update="update" />
-      </template>
-      <p v-if="!updates[teammate]">
-        No updates have been shared.
-      </p>
-    </section>
-    <button class="add-update-btn" @click="goToAddForm">+ Add Update</button>
-  </main>
+  <div class="container">
+    <UpdateSidebar :project="project"/>
+    <main>
+      <h1>
+        Project: {{ project.projectName }}
+      </h1>
+      <section
+        v-for="(updateList, key) in updates"
```

Comment:
just curious, where does this updates come from/get defined?

Reply:
Its meant to be used as a mixin, so the component that imports it should have it defined. In a future PR, I give it a default value just in case